pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.sonarsource.parent</groupId>
     <artifactId>parent</artifactId>
-    <version>71.0.0.1292</version>
+    <version>72.0.0.1405</version>
```

Comment:
I guess the version can be updated to 74.0.0.1768 here as well.

Reply:
Yes, I am waiting for the release of the sonar-scanner-java-library to update this PR